clang/tools/cir-opt/cir-opt.cpp

```diff
@@ -45,9 +46,10 @@ int main(int argc, char **argv) {
     return cir::createConvertCIRToMLIRPass();
   });
 
-  ::mlir::registerPass([]() -> std::unique_ptr<::mlir::Pass> {
-    return cir::direct::createConvertCIRToLLVMPass();
-  });
+  mlir::PassPipelineRegistration<mlir::EmptyPipelineOptions> pipeline(
```

Comment:
here we register the  pipeline with the name `cir-to-llvm`. So we need to rename the pass in `LowerToLLVM.cpp` - otherwise we wouldn't be able to run either of them due to name conflict